master/introduction/network-architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Network Architecture
+---
+
+The following overview is only valid for **container** deployments:
+
+![network architecture](/images/container-networking.layout.png)
+
+Each container is given a veth pair, with one end in the host namespace and the other
```

Comment:
I think we need to pad this out to describe the lifecycle, otherwise the information is dangling.

Did we not have anything in our old docs that basically walked through the container lifecycle?